sdk/sdk-common/src/common/implementation/TokenAmount.ts

```diff
@@ -79,6 +79,10 @@ export class TokenAmount implements ITokenAmount {
     return new BigNumber(this.amount).times(this._baseUnitFactor).toFixed(0)
   }
 
+  public toBaseUnitAsBn(): BigNumber {
```

Comment:
No need to do anything, but I'm a bit worried about this convenience methods as we have `toBaseUnit`, `toBaseUnitAsBn`, `toBN` with kind of inconsistent names and also I'm not sure it is clear how to use them. Ideally we should have only one `toBN()` or `toBaseUnit` as the rest of calculations should be done by using the wrapping type (Price, Percentage, TokenAmount, etc...).  I'll add a story to the SDK backlog to review all this